warplib/maths/mod_signed.cairo

```diff
@@ -0,0 +1,384 @@
+# AUTO-GENERATED
```

Comment:
The warplib gitignore should be used to exclude the autogenerated warplib files from being included in git

Reply:
Warplib ingores are moved to root gitignore file . And, there is already an entry for this in .gitignore. I'll delete this file and will push.